src/pages/ThorChainLP/AvailablePools.tsx

```diff
@@ -106,23 +98,27 @@ const PoolButton = ({ pool }: PoolButtonProps) => {
         </Box>
         <RawText>{pool.name}</RawText>
         <Tag size='sm'>
-          <Amount.Percent value={pool.poolAPY} />
+          <Amount.Percent value={pool.annualPercentageRate} />
```

Comment:
use `annualPercentageRate` instead of `poolAPY` as I have seen `poolAPY` be 0 at times whereas `annualPercentageRate` always seems accurate